pkg/loadbalancer/loadbalancer_test.go

```diff
@@ -432,3 +435,26 @@ func TestServiceFlags_String(t *testing.T) {
 		})
 	}
 }
+
+type ConstSuite struct{}
+
+var _ = check.Suite(&ConstSuite{})
+
+// ConstTest checks whether constants of the C and Go
+// structs for the loadbalancer match.
+//
+// C struct size info is extracted from the given ELF object file .rodata section.
+func (t *ConstSuite) TestConst(c *check.C) {
+	// Validate alignments of C and Go equivalent structs
```

Comment:
nit: This comment talks about alignments and structs, but the constchecker doesn't appear to deal with these things.